src/js/components/modals/AddRepositoryFormModal.js

```diff
@@ -60,6 +60,11 @@ class AddRepositoryFormModal extends mixin(StoreMixin) {
 
   getWithinRangeFunction(min, max) {
```

Comment:
How about using the shine new [`isNumberInRange`](https://github.com/dcos/dcos-ui/blob/master/src/js/utils/ValidatorUtil.js#L48) in combination with [`!isDefined`](https://github.com/dcos/dcos-ui/blob/master/src/js/utils/ValidatorUtil.js#L2) instead? That would make it way easier to under the validation rule.